src/components/content/content.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Person } from '../../types/Person';
+
+interface Props {
+  peopleFromServer: Person[];
+  setSelectedPerson: React.Dispatch<React.SetStateAction<'' | Person>>;
+}
+
+const Content: React.FC<Props> = ({ peopleFromServer, setSelectedPerson }) => {
+  const find = (person: Person | '') => {
```

Comment:
The `find` function accepts a parameter of type `Person | ''`. Since the `peopleFromServer` array contains only `Person` objects, the parameter type could be simplified to `Person` for consistency and type safety. Consider updating the parameter type to `Person`.